src/sagemaker/jumpstart/model.py

```diff
@@ -309,11 +315,12 @@ def _is_valid_model_id_hook():
         self.tolerate_vulnerable_model = model_init_kwargs.tolerate_vulnerable_model
         self.tolerate_deprecated_model = model_init_kwargs.tolerate_deprecated_model
         self.region = model_init_kwargs.region
-        self.model_package_arn = model_init_kwargs.model_package_arn
         self.sagemaker_session = model_init_kwargs.sagemaker_session
 
         super(JumpStartModel, self).__init__(**model_init_kwargs.to_kwargs_dict())
 
+        self.model_package_arn = model_init_kwargs.model_package_arn
```

Comment:
why moved it to below?

Reply:
we are calling the Model's constructor here - `super(JumpStartModel, self).__init__(**model_init_kwargs.to_kwargs_dict())` which is resetting the value of model_package_arn to None

Reply:
so moved it below so that the value isnt reset